lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -336,6 +335,7 @@ def add_block(xblock):
             blocks[unicode(xblock.location)] = {
                 u'id': unicode(xblock.location),
                 u'type': xblock.category,
+                u'parent': None,
```

Comment:
it will not set `None` when we request data for certain `block_types` and we are testing those cases in `tests_api.py` where `parent` is set a proper `usage_id`